src/main/java/io/anserini/index/IndexWebCollection.java

```diff
@@ -101,15 +117,21 @@ private int indexWarcRecord(WarcRecord warcRecord) throws IOException {
       // document ID
       document.add(new StringField(FIELD_ID, id, Field.Store.YES));
 
-      // entire document
-      if (positions)
-        if (docVectors)
-          document.add(new TermVectorsTextField(FIELD_BODY, contents));
-        else
-          document.add(new TextField(FIELD_BODY, contents, Field.Store.NO));
-      else
-        document.add(new NoPositionsTextField(FIELD_BODY, contents));
-
+      FieldType fieldType = new FieldType();
+      fieldType.setStored(true);
+      fieldType.setStoreTermVectors(true);
+
+        // entire document
+      if (positions) {
+          // Important, lucene 5 no longer has simple setIndexed option
+          // set through index options
+          fieldType.setIndexOptions(IndexOptions.DOCS_AND_FREQS_AND_POSITIONS);
+          fieldType.setStoreTermVectorPositions(true);
+          document.add(new Field(FIELD_BODY, contents, fieldType));
+      } else {
+          fieldType.setIndexOptions(IndexOptions.DOCS_AND_FREQS);
+          document.add(new Field(FIELD_BODY, contents, fieldType));
+      }
```

Comment:
Oh thanks! Read the field options and thought that was all there was. Also found the NoPositionsTextField in the Anserini's Index package.
I remember though at the beginning when we were trying to use term vectors only specifying through the IndexOption actually stored the tf files. Am I mixing up some concepts?

I've updated the code.